tests/conftest.py

```diff
@@ -267,3 +268,109 @@ def additional_test_data():
         "outcome_programme": outcome_programme,
         "outcome_no_projects": programme_outcome2,
     }
+
+
+@pytest.fixture
+def valid_review_and_sign_off_section():
+    valid_review_and_sign_off_section = {
+        "8 - Review & Sign-Off": pd.DataFrame(
+            {
+                0: ["", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""],
+                1: [
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "Name",
+                    "Role",
+                    "Signature",
+                    "Date",
+                    "",
+                    "",
+                    "",
+                    "Name",
+                    "Role",
+                    "Signature",
+                    "Date",
+                    "",
+                ],
+                2: [
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "John",
+                    "Head of Tree Planting",
+                    "Image1",
+                    "01/01/2260",
+                    "",
+                    "",
+                    "",
+                    "John",
+                    "Head of Tree Planting",
+                    "Image2",
+                    "01/01/2260",
+                    "",
+                ],
+            }
+        ),
+    }
+
+    return valid_review_and_sign_off_section
+
+
+@pytest.fixture
+def invalid_review_and_sign_off_section():
+    invalid_review_and_sign_off_section = {
+        "8 - Review & Sign-Off": pd.DataFrame(
+            {
+                0: ["", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", "", ""],
+                1: [
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "Name",
+                    "Role",
+                    "Signature",
+                    "Date",
+                    "",
+                    "",
+                    "",
+                    "Name",
+                    "Role",
+                    "Signature",
+                    "Date",
+                    "",
+                ],
+                2: [
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "",
+                    "Head of Tree Planting",
+                    "Image1",
+                    "01/01/2260",
+                    "",
+                    "",
+                    "",
+                    "John",
+                    "",
+                    "Image2",
+                    "",
+                    "",
+                ],
+            }
+        ),
+    }
+
+    return invalid_review_and_sign_off_section
```

Comment:
I don't think these belong here. This file is best used to store fixtures that are used across the whole of the test package. These are very specific to just the round 4 validation. Maybe it's best we split the round 4 specific validation out into its own package within validation tests and then create a `mock_data.py` alongside the existing `test_tf_r4_specific_validations.py` module

Reply:
Implemented the above